### PR TITLE
Update repository formatting/Apply ESLint

### PR DESCRIPTION
**Pull request by sigmachirality**
We should talk sometime about the code, it's kinda cringe ngl

### CODE REVIEW

File: remi/monster.js [hunk 1]
```diff
@@ -1,123 +1,122 @@
...
+  return roll
 }
 
 // Read in the monsters, segregate, and store
-const threeStarFile = 'monsters/three_star_drops.txt';
-const fourStarFile = 'monsters/four_star_drops.txt';
-const fiveStarFile = 'monsters/five_star_drops.txt';
-const fiveStarGodsFile = 'monsters/five_star_gods.txt';
-const GFEFile = 'monsters/GFE.txt';
+const threeStarFile = 'monsters/threeStarDrops.txt'
```
Comment (AndrooTheChen) [inline remi/monster.js:46]: These are filenames in directory /monsters, please change these as well.

Comment (sigmachirality) [reply remi/monster.js:46]: nah dude change back the filenames, only the variable names matter

File: remi/commands.js [hunk 2]
```diff
@@ -1,166 +1,167 @@
...
-    msg.channel.send(`**%roll** - roll for a monster!\n` + 
-                                `**%help** - list commands\n` +
-                                `**%monbox** - print your monster box\n` +
-                                `**%myrolls** - print your rolls.\n` +
-                                `**%myclaims** - print your claims.\n\n` +
-                                `__***FAQ***__\n` +
-                                `You have **60** seconds to claim a monster from when it is rolled.\n` +
-                                `You may roll up to **6** times every **45** minutes\n` +
-                                `You may claim up to **2** monsters every **45** minutes\n`);
+function help (msg) {
```
[the remark this answers is not in the document]
Comment (sigmachirality) [reply remi/commands.js:155]: You can, I'm just lazy

File: remi/commands.js [hunk 3]
```diff
@@ -1,166 +1,167 @@
...
-};
+  claim,
+  claimid,
+  exec,
+  help,
+  monbox,
+  myrolls,
+  myclaims,
+  roll
+}
```
[the remark this answers is not in the document]
Comment (sigmachirality) [reply remi/commands.js:18]: I used standardjs formatting, you can ask them here https://standardjs.com/

File: remi/rutil.js [hunk 4]
```diff
@@ -49,26 +38,17 @@ function printTimeStamp (time) {
...
-  let str = '```\n'
-
-  monBox.forEach((monster) => {
-    str = str + `${monster}\n`
-  })
-
-  str = str + '```'
-  return str
-}
+const monPrint = monBox => `\`\`\`
```
Comment (AndrooTheChen) [inline remi/rutil.js:49]: good stuff

File: remi/rutil.js [hunk 5]
```diff
@@ -49,26 +38,17 @@ function printTimeStamp (time) {
  * @return {string} current time stamp in readable format
  */
 function getTimeStamp () {
-  const today = new Date()
-  const date = (today.getUTCMonth() + 1) + '/' + today.getUTCDay() + '/' + today.getFullYear()
-  const time = today.getUTCHours() + ':' + today.getUTCMinutes() + ':' + today.getUTCSeconds()
-  return `${date} ${time}`
+  const {getUTCMonth, getUTCDay, getFullYear, getUTCHours, getUTCMinutes, getUTCSeconds} = new Date()
```
Comment (AndrooTheChen) [inline remi/rutil.js:41]: I'm getting a strange error with getUTCMonth, for some reason it's not recognized as a Date object? Are we calling it wrong here or missing a library or something?

Output when running:
```
(node:18046) UnhandledPromiseRejectionWarning: TypeError: this is not a Date object.
    at getUTCMonth (<anonymous>)
    at getTimeStamp (/home/androo1/Workspace/REMi/remi/rutil.js:42:13)
    at Object.err (/home/androo1/Workspace/REMi/remi/rutil.js:92:19)
    at Object.connectDB (/home/androo1/Workspace/REMi/remi/mongoUsers.js:77:11)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:18046) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:18046) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:18046) UnhandledPromiseRejectionWarning: TypeError: this is not a Date object.
    at getUTCMonth (<anonymous>)
    at getTimeStamp (/home/androo1/Workspace/REMi/remi/rutil.js:42:13)
    at Object.log (/home/androo1/Workspace/REMi/remi/rutil.js:58:19)
    at Client.<anonymous> (/home/androo1/Workspace/REMi/remi/remi.js:36:9)
    at Client.emit (events.js:315:20)
    at WebSocketManager.triggerClientReady (/home/androo1/Workspace/REMi/node_modules/discord.js/src/client/websocket/WebSocketManager.js:433:17)
    at WebSocketManager.checkShardsReady (/home/androo1/Workspace/REMi/node_modules/discord.js/src/client/websocket/WebSocketManager.js:417:10)
    at WebSocketShard.<anonymous> (/home/androo1/Workspace/REMi/node_modules/discord.js/src/client/websocket/WebSocketManager.js:199:14)
    at WebSocketShard.emit (events.js:315:20)
    at WebSocketShard.checkReady (/home/androo1/Workspace/REMi/node_modules/discord.js/src/client/websocket/WebSocketShard.js:467:12)
(node:18046) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

Comment (sigmachirality) [reply remi/rutil.js:41]: oh I get it, the js date object implementation is shit code. Replace const {getUTCMonth ... etc } with const date = new Date() and then add date. to the front of each method call in the format string

**Pull request merged**